src/backend/api/v1/clusters/views.py

```
@@ -0,0 +1,181 @@
+from backend.apps.aap_auth.authentication import AAPAuthentication
+from backend.apps.clusters.encryption import encrypt_value
+from backend.apps.clusters.models import Cluster, JobLaunchTypeChoices
+from backend.apps.scheduler.models import SyncJob, JobTypeChoices
+
+from .serializers import ClusterReadSerializer, ClusterWriteSerializer
+
+logger = logging.getLogger('automation_dashboard.api.clusters')
+
+
+class ClusterListCreateView(APIView):
+    authentication_classes = [AAPAuthentication]
```

Ruff (0.15.13)

[warning] Mutable default value for class attribute

(RUF012)